src/main/java/Event.java

```diff
@@ -0,0 +1,12 @@
+public class Event extends Task{
+    private String at;
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
+    }
+
+    @Override
+    public String toString(){
+        return "[E]" + super.toString() + " (at: " + at + ")";
+    }
```

Comment:
I like that you made it clear your are overriding a method (in this case in the object class).